src/utils/update.change.link.ts

```diff
@@ -0,0 +1,11 @@
+export const createUpdateChangeLinkConfig = (basePath: string, id: string, text: string, dataEventId: string) => {
```

Comment:
Remove Update and change name to createChangeLinkConfigForPage()
file name too as now used not just for update